helm/logging-app-chart/templates/fluentbit/daemonset.yaml

```
@@ -0,0 +1,110 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Values.fluentbit.name }}
+  namespace: {{ .Release.namespace }}
+  labels:
+    app: {{ .Values.fluentbit.name }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.fluentbit.name }}
+  template:
+    metadata:
+      annotations:
+        releasetime: {{ $.Release.Time }}
+      labels:
+        app: {{ .Values.fluentbit.name }}
+    spec:
+      tolerations:
+      # Tolerate master taint
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      serviceAccountName: {{ .Values.fluentbit.name }}
+      securityContext:
+        runAsUser: {{ .Values.fluentbit.userID }}
+        runAsGroup: {{ .Values.fluentbit.groupID }}
+      containers:
+      - name: {{ .Values.fluentbit.name }}
+        image: {{ .Values.fluentbit.image.registry }}/{{ .Values.fluentbit.image.name }}:{{ .Values.fluentbit.image.tag }}
```

```suggestion
        image: {{ .Values.image.registry }}/{{ .Values.fluentbit.image.name }}:{{ .Values.fluentbit.image.tag }}
```

All the images should use `image.registry` as this is the only value we set in the app catalog configmap,

---

ahh ok got it

---

but then I need to remove the image from child values parameter, right?

---

and image to the top values yaml?

---

Its just image.registry this applies to.

I'm fine with `.Values.fluentbit.image.name` unless you think having all the images under `.Values.image` is better?